homeassistant/components/sensor/tibber.py

```diff
@@ -171,8 +171,15 @@ def __init__(self, tibber_home):
 
     async def _async_callback(self, payload):
         """Handle received data."""
-        data = payload.get('data', {})
-        live_measurement = data.get('liveMeasurement', {})
+        errors = payload.get('errors')
+        if errors:
+            _LOGGER.error(errors[0])
+        data = payload.get('data')
```

Comment:
Will there be data if an error occurred?

Reply:
Probably not